includes.php

```diff
@@ -14,12 +14,15 @@
 	$config = array_merge( $config, $localConfig );
 }
 
+OOUI\Theme::setSingleton( new OOUI\WikimediaUITheme() );
+
 echo '<!DOCTYPE html>
 <html>
 	<head>
 		<meta charset="utf-8">
 		<title>Patch demo</title>
 		<link rel="stylesheet" href="index.css">
+		<link rel="stylesheet" href="node_modules/oojs-ui/dist/oojs-ui-wikimediaui.min.css">
```

Comment:
we don't deploy with `npm i` at the moment, so perhaps this file should just be copied to a lib folder?

Reply:
Adding `npm install --production` to the deploy step is probably the easiest.